crates/ingress-grpc/src/reflection/v1alpha.rs

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2023 -  Restate Software, Inc., Restate GmbH.
```

Comment:
I think there's a way we can avoid copying, but it's fine for the time being. perhaps open an issue to do that, i can take it at a later point in time.

Reply:
Alright, will do.

Reply:
https://github.com/restatedev/restate/issues/1192